kyuubi-server/src/main/resources/sql/derby/statestore-schema.derby.sql

```diff
@@ -0,0 +1,47 @@
+--
+-- Licensed to the Apache Software Foundation (ASF) under one or more
+-- contributor license agreements.  See the NOTICE file distributed with
+-- this work for additional information regarding copyright ownership.
+-- The ASF licenses this file to You under the Apache License, Version 2.0
+-- (the "License"); you may not use this file except in compliance with
+-- the License.  You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+--
+
+CREATE TABLE batch_metadata(
+    key_id bigint PRIMARY KEY GENERATED BY DEFAULT AS IDENTITY,
+    batch_id varchar(36) NOT NULL,
+    batch_owner varchar(1024) NOT NULL,
+    ip_address varchar(512),
+    session_conf clob,
+    kyuubi_instance varchar(1024) NOT NULL,
```

Comment:
does the instance never change?

Reply:
yes, it was never change.

Reply:
it means kyuubi_instance stop and never come back, the record becomes unhandled?

Reply:
we can enhance it in follow up